### PR TITLE
Hyperscale citus workbook onboarding

### PR DESCRIPTION
**Pull request by pradeepkintali**
## PR Checklist
image

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
This PR is to onboard Hyperscale Citus Workbook template for Metrics Overview. Please verify it here
https://portal.azure.com/?feature.customportal=false&forceEnableFeatures=Microsoft.DBforPostgreSQL%2FmarlinWorkbooks&feature.workbookGalleryRedirect=https%3A%2F%2Fprkintpackage.blob.core.windows.net%2Ftemplates%2Fpackage

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Hyperscale Citus/Metrics Overview/Metrics Overview.workbook
Comment (gardnerjr) [inline Workbooks/Hyperscale Citus/Metrics Overview/Metrics Overview.workbook:203]: nit: FYI, accessibility reviews would complain that "click here" isn't a useful link.  makes more sense to make the "active" part of the sentence "learn more about useful diagnostic queries" as the actual link text

**Review by gardnerjr — APPROVED** [review]
approved from the codeowners side.  didn't dig into the details of the workbook, team owners should review and approve the actual content there.

**Review by akashraokm — APPROVED** [review]

**Pull request merged**
